### PR TITLE
Handle tuple values in Daml ledger export

### PR DESCRIPTION
**Pull request by aherrmann-da**
Encodes tuple values using tuple syntax.

### Pull Request Checklist

- [x] Read and understand the [contribution guidelines](https://github.com/digital-asset/daml/blob/main/CONTRIBUTING.md)
- [x] Include appropriate tests
- [x] Set a descriptive title and thorough description
- [ ] Add a reference to the [issue this PR will solve](https://github.com/digital-asset/daml/issues), if appropriate
- [x] Include changelog additions in one or more commit message bodies between the `CHANGELOG_BEGIN` and `CHANGELOG_END` tags
- [ ] Normal production system change, include purpose of change in description

NOTE: CI is not automatically run on non-members pull-requests for security
reasons. The reviewer will have to comment with `/AzurePipelines run` to
trigger the build.

### CODE REVIEW

File: daml-script/export/src/main/scala/com/daml/script/export/Encode.scala [hunk 1]
```diff
@@ -142,9 +142,14 @@ private[export] object Encode {
       cidMap: Map[ContractId, String],
       v: Value.Sum,
   ): Doc = {
+    def isTupleRecord(recordId: Identifier): Boolean = {
```
Comment (cocreature) [inline daml-script/export/src/main/scala/com/daml/script/export/Encode.scala:145]: given that these are all in a stable package, maybe worth checking the package id as well?

File: daml-script/export/src/test/scala/com/daml/script/export/EncodeValueSpec.scala [hunk 2]
```diff
@@ -50,6 +50,31 @@ class EncodeValueSpec extends AnyFreeSpec with Matchers {
         |    c = 42
         |  c = M.R3""".stripMargin.replace("\r\n", "\n")
     }
+    "tuple" in {
+      def tupleId(n: Int): v.Identifier = v
+        .Identifier()
+        .withPackageId("40f452260bef3f29dede136108fc08a88d5a5250310281067087da6f0baddff7")
```
Comment (cocreature) [inline daml-script/export/src/test/scala/com/daml/script/export/EncodeValueSpec.scala:56]: can we get the package id from daml-lf/language?

**Review by cocreature — APPROVED** [review]
Thanks!

File: daml-script/export/src/main/scala/com/daml/script/export/Encode.scala (hunk 1 above)
Comment (aherrmann-da) [reply daml-script/export/src/main/scala/com/daml/script/export/Encode.scala:144]: Done.

File: daml-script/export/src/test/scala/com/daml/script/export/EncodeValueSpec.scala (hunk 2 above)
Comment (aherrmann-da) [reply daml-script/export/src/test/scala/com/daml/script/export/EncodeValueSpec.scala:56]: We could, but the lookup key is `daml-prim-DA-Types-40f452260bef3f29dede136108fc08a88d5a5250310281067087da6f0baddff7`. So, we'd still end up hard-coding the hash.

Comment (cocreature) [reply daml-script/export/src/test/scala/com/daml/script/export/EncodeValueSpec.scala:56]: ah fair enough that’s not very helpful

**Pull request merged**